ccron/include/ccron/tick.hpp

```diff
@@ -28,11 +28,14 @@ struct Tick {
   // Time that is the same across replicas after being agreed upon.
   // It is optional as the time service that provides agreement across replicas is itself optional.
   std::optional<std::chrono::milliseconds> ms_since_epoch;
+
+  // a sequnce number, may contain several ticks, this field is used in order to order them.
+  std::uint64_t internal_order{};
 };
 
 inline bool operator==(const Tick& l, const Tick& r) {
```

Comment:
You mentioned in the description that we will total order the ticks. Don't we need some form of operator< for this?

Reply:
the ordering is done against the Batch cmf msg that is saved in the reserved pages and saves in addition to the sequence number and internal order also the target block id.